utils/node_local_test.py

```diff
@@ -877,7 +902,12 @@ def fetch_pools(self):
     def _make_pool(self):
         """Create a DAOS pool"""
 
-        size = 1024*2
+        # If running as a small system with tmpfs already mounted then this is likely a docker
+        # container so restricted in size.
+        if self.conf.args.no_root:
+            size = 1024*2
+        else:
+            size = 1024*4
```

Comment:
FYI, new option:
```
$ dmg pool create --help
...
-z, --size=         Total size of DAOS pool or its percentage ratio (auto)
```
Though, if you're creating multiple pools it's a little tricky.

Reply:
How is this different from what this code does?

Reply:
I mean you can do something like `dmg pool create --size 50%` to avoid switching between `1024*2` and `1024*4`. Since `50%` would be relative to how much space is actually available.

Reply:
I don't think it would help.  GHA has very small memory available so it wants 50% of space, NLT when run on Jenkins is on a physical node so it probably only needs 5%.